public/javascripts/SVLabel/spec/SVLabel/modal/ModalMissionComplete-spec.js

```diff
@@ -112,17 +175,16 @@ describe("ModalMissionComplete", function () {
         neighborhood = new NeighborhoodMock();
         neighborhood.properties.name = "Test Neighborhood";
 
-        modalMissionMock = new ModalMissionMock(modalModel)
     });
 
-    describe("`show` method", function () {
-        it("should open a modal window", function () {
-            modalMissionComplete.hide();
+    describe("`hide` method", function () {
+        it("should hide a modal window", function () {
+            modal.hide();
             expect(uiModalMissionComplete.holder.css('visibility')).toBe('hidden');
             expect(uiModalMissionComplete.foreground.css('visibility')).toBe('hidden');
             expect(uiModalMissionComplete.background.css('visibility')).toBe('hidden');
 
-            modalMissionComplete.show(mission, neighborhood);
```

Comment:
it was like that in you last spec file before you split the modules further. I changed it to modalMissionComplete now